### PR TITLE
feat(components): add List component

### PR DESCRIPTION
**Pull request by akhar**
WIP.  Add va-list, va-list-item and va-list-item-tile components.

</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (a non-breaking change that doesn't add any feature but makes things better)

### CODE REVIEW

File: src/vuestic-theme/vuestic-components/va-list/VaList.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="va-list">
+    <div class="va-list-item-header">I need a header</div>
+    <va-list-item
+      v-for="(item, index) of items"
+      :dividers="dividers"
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/va-list/VaList.vue]: Divider should probably implemented as separate component (see vuetify)

File: src/vuestic-theme/vuestic-components/va-list/VaList.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      :index="index"
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/va-list/VaList.vue]: Not sure what is that for.

File: src/vuestic-theme/vuestic-components/va-list/VaList.demo.vue [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<template>
+  <VbDemo>
+    <div class="va-list-demo">
+      <VaList :items="items" :dividers="true" @item-click="onClick"/>
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/va-list/VaList.demo.vue]: Exact design is a requirement.

image

**Comment by jean-moldovan** [comment]
Closing in favor of: https://github.com/epicmaxco/vuestic-admin/pull/607

**Pull request closed without merging**